toolchain/sem_ir/interface.h

```diff
@@ -20,6 +20,12 @@ struct Interface : public Printable<Interface> {
   // until we reach the `}` of the interface definition.
   auto is_defined() const -> bool { return defined; }
 
+  // Determines whether we're currently defining the interface. This is true
+  // between the braces of the interface.
+  auto is_being_defined() const -> bool {
+    return definition_id.is_valid() && !defined;
```

Comment:
```suggestion
    return definition_id.is_valid() && !is_defined();
```

Just to enforce the relationship.